app/(chat)/api/chat/route.ts

```
@@ -44,106 +48,250 @@ export async function POST(request: Request) {
     } = await request.json();
 
     const session = await auth();
-    const baasSession = await meetingBaas.auth();
-
-    if (!session || !session.user || !session.user.id) {
-      return new Response('Unauthorized', { status: 401 });
-    }
+    let baasSession = await meetingBaas.auth();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'baasSession' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)